Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -794,6 +795,16 @@ void VertexManagerBase::CalculateBinormals(NativeVertexFormat* format)
     vertex_shader_manager.constants.cached_binormal = VertexLoaderManager::binormal_cache;
     vertex_shader_manager.dirty = true;
   }
+
+  if (vert_decl.normals[0].enable)
+    return;
+
+  VertexLoaderManager::normal_cache[0] = 0;
```

Comment:
```suggestion
  VertexLoaderManager::normal_cache[3] = 0;
```

not caught by tests since the test only works with the vertex loader, not with the vertex manager